webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -1,23 +1,35 @@
 <template>
-  <ds-button v-if="totalNotifications <= 0" class="notifications-menu" disabled icon="bell">
-    {{ totalNotifications }}
-  </ds-button>
+  <div
+    v-if="totalNotifications <= 0"
```

Comment:
Why do we have to add this extra `v-if` here? :thinking:

Reply:
there is a `v-if` because there is the additional functionality that the `NotificationsMenu`, if clicked on when there are no notifications, updates the notifications

Reply:
Also, @Tirokk, I submit as proof I didn't purposely remove the controversial line of code with comments to explain to the `PR` reviewer. 
Do you remember when we discussed and I changed this line of code to be `totalNotifications === 0`??
Must not have saved the file, or some other quirk with `git`

Reply:
The extra `div` with this `v-if` is to switch between the disabled and enabled button and have the ability to update it on click even if disabled. Like @mattwr18 explained.